misc/hooks/pre-commit-clang-format

```diff
@@ -97,7 +97,7 @@ do
     # The sed call is necessary to transform the patch from
     #    --- $file timestamp
     #    +++ - timestamp
-    # to both lines working on the same file and having a a/ and b/ prefix.
+    # to both lines working on the same file and having a/ and b/ prefix.
```

Comment:
That was not a typo, it's called "**a** prefix" :)
But well, doesn't matter much, and the correct here would likely be a plural anyway.